Week1/subway/index.html

```diff
@@ -0,0 +1,65 @@
+<html>
+  <head></head>
+  <style>
+	#map {
+	   height: 100%;
+	 }
+  </style>
+  <body>
+	<div id="map"></div>
+  </body>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  <script>
+	let map;
+	function getLatLng(data){
+		//entrances
+		const entrances = data[0];
+		let entlocations = [];
+		for(let e of entrances){
+			entlocations.push(e["the_geom"]);
+		}
+		//stations
+		const stations = data[1];
+		let stalocations = [];
+		for(let s of stations){
+			stalocations.push(s["the_geom"]);
+		} // I know there is a better way to do this, but I forgot !
+		return [entlocations, stalocations];
+	}
+	function getData(callback){
+	  $.ajax({
+			url: "http://localhost:8000/subways.json",
+			method: "GET",
+			datatype: "json",
+			async: false,
+			success : (data) => {
+				if(typeof callback == "function"){
+					callback(data);
+				}
+			}
+	  });
+	}
+	function makeMarkers(data){
+		const latlng = getLatLng(data);
+		console.log(latlng);
+		for(let subway of latlng){
```

Comment:
I believe these actually work as `const` as well since by the time the loop gets back to this line, it has already dropped (unallocated from memory) the previous variable.